src/Cake.Scripting.Abstractions/Cake.Scripting.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
@bjorkstromm can you provide more details as to exactly why this is required, as there is some confusion within the team.  Thanks

Reply:
This is a library consumed by OmniSharp. OmniSharp targets `net4.8`. So it can't possibly consume a `net6.0` library. Keep the `netstandard2.0` moniker.

Reply:
See
https://github.com/OmniSharp/omnisharp-roslyn/blob/605bb3669b16275d9f57a46d1e05bf4d0941b197/src/OmniSharp.Cake/OmniSharp.Cake.csproj#L18